Chore/update angular v17 10 provide a login component using appwrite

mathisvester opened the pull request.

santoshyadavdev commented:
interesting just to understand you create a temp branch from main and merging it back to this feature branch with some extra changes right?

mathisvester commented:
@santoshyadavdev I created a temp branch from this feature branch and merged it with main branch which is already on v17. Then I updated NgRx to v17 and some other packages to support new control flow for example and adjusted the code accordingly.
